app/views/playlists/edit.html.erb

```diff
@@ -16,10 +16,10 @@
 
         <%= render 'add_greenfield_downloads' if allow_greenfield_playlist_downloads?(@user) %>
 
-        <%= simple_form_for [@user, @playlist] do |f| %>
-          <%= f.input :title %>
-          <%= f.input :year, input_html: {maxlength: 4, size: 4} %>
-          <%= f.input  :description %>
+        <%= form_for [@user, @playlist] do |f| %>
```

Comment:
I thought that `form_tag` and `form_for` are soft deprecated since Rails 5... 1? 
Was there a reason for using `form_for` here?

Reply:
form_for isn't yet deprecated, but you are right — if this was a template I expected to last more than another couple months, it should be changed.